OntologyMapper/src/Mappings/Mapped/Json/v0/BrickRec/mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json

```diff
@@ -5480,15 +5480,15 @@
     {
       "OutputDtmiFilter": "*",
       "OutputPropertyName": "externalIds",
-      "InputPropertyName": "mappingKey",
+      "InputPropertyNames": [ "mappingKey" ],
```

Comment:
These changes seem unrelated to Adding Nuget Packages to Azure Artifacts. Are they intended to be added?